src/plugins/unified_histogram/public/chart/histogram.tsx

```diff
@@ -204,6 +230,10 @@ export function Histogram({
     }
   `;
 
+  if (!checkValidDataViewConfig(dataView, visContext, lensProps.attributes.state.adHocDataViews)) {
+    return <></>;
```

Comment:
@jughosta wo we have an alternative here? we need to prevent the rendering for the flaky case, should we render something, given we move forward with this PR (it woulld need to be investigated or refactored later on, since the given state causing the flakiness ideally should not happen)

Reply:
I don't have an alternative right now. What I've tried was causing other test failures and extra fetches. So let's go with your solution.

Reply:
Ok, so we should open an issue to remove this code, given we resolve the root cause, which might be a bigger refactoring